templates/tfengine/components/terraform/main/main.tf

```diff
@@ -18,7 +18,12 @@ terraform {
     google      = "~> 3.0"
     google-beta = "~> 3.0"
   }
-  backend "gcs" {}
+  backend "gcs" {
+    {{- if not (get . "disable_gcs_backend_config")}}
```

Comment:
"not get disabled" is a double negative. Why not replace with "if get . "enable_..."?

Reply:
Then all users will always need to pass in enable_gcs_backend. In this case, if no field is passed then this block will be enabled.

We had an enabled helper before but given this field is only set by internal templates I don't think double negative is that big a deal. External users aren't expected to be passing this field.